content/en/docs/setup/windows/user-guide-windows-nodes.md

```diff
@@ -130,7 +130,7 @@ Once you have a Linux-based Kubernetes master node you are ready to choose a net
     kubectl apply -f kube-flannel.yml
     ```
 
-    Next, since the Flannel pods are Linux-based, apply a NodeSelector patch, which can be found [here](https://github.com/Microsoft/SDN/blob/1d5c055bb195fecba07ad094d2d7c18c188f9d2d/Kubernetes/flannel/l2bridge/manifests/node-selector-patch.yml), to the Flannel DaemonSet pod:
```

Comment:
Just a thought, should this reference a specific commit, in case the file gets moved? I don't think this node-selector-patch.yaml requires any maintenance